dialogue-target/src/main/java/com/palantir/dialogue/ConjureErrors.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue;
+
+import com.palantir.logsafe.Safe;
+
+public final class ConjureErrors {
+    private ConjureErrors() {}
+
+    public abstract static class BaseEndpointError<T> {
```

Comment:
I don't think the `ConjureErrors` type buys us anything here, can we remove that and make `BaseEndpointError` the top-level class? Might consider renaming it to `EndpointError`, I don't think `Base` adds a ton